packages/blocks/src/page-block/edgeless/components/image/edgeless-image.ts

```diff
@@ -0,0 +1,57 @@
+import { WithDisposable } from '@blocksuite/lit';
+import { LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { html } from 'lit/static-html.js';
+
+import type { ImageBlockModel } from '../../../../image-block/index.js';
+import { Bound } from '../../../../surface-block/index.js';
+import type { SurfaceBlockComponent } from '../../../../surface-block/surface-block.js';
+
+@customElement('edgeless-image')
```

Comment:
Now we have  both `edgeless-image` and `affine-edgeless-image` in the codebase, which is duplicate. How to differentiate them?